app/src/main/java/com/asfoundation/wallet/iab/presentation/IABButtons.kt

```diff
@@ -0,0 +1,98 @@
+package com.asfoundation.wallet.iab.presentation
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.asfoundation.wallet.iab.theme.IAPTheme
+import kotlin.random.Random
+
+@Composable
+fun IABOpaqueButton(
+  modifier: Modifier = Modifier,
+  text: String,
+  isEnabled: Boolean = true,
+  onClick: () -> Unit,
+) {
+  Button(
+    onClick = onClick,
+    enabled = isEnabled,
+    modifier = modifier
+      .fillMaxWidth()
+      .height(40.dp),
+    border = BorderStroke(0.dp, Color.Transparent),
+    shape = RoundedCornerShape(50),
+    colors = ButtonDefaults.buttonColors(
```

Comment:
I think it's a good idea to already include the .testTag("") that Tiago wants for test the buttons, avoiding future work with that screen